pgmpy/factors/CPD.py

```diff
@@ -11,6 +12,7 @@
 from pgmpy.extern import tabulate
 from pgmpy.extern import six
 from pgmpy.extern.six.moves import range, zip
+from warnings import warn
```

Comment:
@ankurankan Yes, about this, actually there is one more issue you might have noticed. Due to carelessness I created a branch that had previous commits with regards to other PRs I had sent and failed to create a fresh branch. Although I have only kept the features pertaining to this particular feature, I missed this line. If you want I could start afresh on a new branch, that doesn't have irrelevant commit history.